tools/2d_simple_filter/filter.xml

```diff
@@ -1,5 +1,12 @@
-<tool id="ip_filter_standard" name="Filter Image" version="0.0.3">
-    <description>applies a standard filter to an image</description>
+<tool id="ip_filter_standard" name="Filter 2D image" version="0.0.3-2">
+    <description>with scikit-image</description>
+    <edam_operations>
+        <edam_operation>operation_3443</edam_operation>
+    </edam_operations>
+    <xrefs>
+        <xref type="bio.tools">scikit-image</xref>

```

Comment:
Because you have this here, Galaxy will look into bio.tools and assign the edam term / operations to the tool.

![grafik](https://github.com/BMCV/galaxy-image-analysis/assets/469983/d7589ff7-acbf-4143-8a0c-a8976074b326)

Essentially it makes adding the ` <edam_operations>` unnecessary.

This is only true if you have a bio.tools entry ...

Reply:
https://bio.tools/scikit-image

Reply:
In an ideal world, all those tools will have at some point a bio.tools entry - which we are currently renaming to `research.tools`. I hope and think - because EDAM terms are hard to handle for tool devs - that we only need to add them manually when we don't have a bio.tools handle.

I'm just a reviewer here and I don't want to screw all your decisions that you have taken, so feel free to kepp it :)

Reply:
I see, the situation is not ideal. But it is an improvement, maybe a bit of an overshoot. I guess I will stick to what we had agreed then for the moment :) we can still revise this decision in the future.